platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -81,6 +91,9 @@ struct odp_crypto_generic_session_t {
 		crypto_func_t func;
 	} auth;
 	unsigned int idx;
+
+	/* Armv8 cipher constants */
+	armv8_cipher_constants_t cc;
```

Comment:
It is not useful to have a comment that merely repeats the type name. The comment should be either improved or removed.

Reply:
I will remove the comment. Thanks